### DISCUSSION TITLE
Execute CreateCompletion() ,then exit unexpectedly

### DISCUSSION THREAD

**Discussion by GrowingHacker** [betalgo/openai#193] [Q&A] [answered]
When this piece of code is executed, it is accidentally exited, there is no error message, and the try catch is useless:
   completionResult = await openAIService.Completions.CreateCompletion(new CompletionCreateRequest()
                {
                    Prompt = "Once upon a time",
                    Model = Models.ChatGpt3_5Turbo
                 });

**Comment by kayhantolga** [comment]
I don't think CreateCompletion works with ChatGpt3_5Turbo Model. Please try to use a different model or [chat ](https://github.com/betalgo/openai/wiki/Chat-GPT) endpoint.

**Reply by GrowingHacker** [reply]
The same goes for using DaVinci3，Should I try the new version?

**Reply by GrowingHacker** [reply]
Oh ! I get it! It's async in the trickster, the main method doesn't wait for it！

**Comment by GrowingHacker** [comment, marked as answer]
It's the async causing trouble,Don't let the main method close soon ,make it  to do something just like :
 Console.ReadLine()  OR   Console.ReadKey()
OR Task.WaitAll(Task.Delay(1000));
